test/jasmine/assets/create_graph_div.js

```diff
@@ -5,6 +5,11 @@ module.exports = function createGraphDiv() {
     gd.id = 'graph';
     document.body.appendChild(gd);
 
+    var closedCaptions = document.createElement('div');
```

Comment:
I don't think we need this since we're not editing the jasmine tests in this PR?